.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 node_js:
 - 'node'
-- '7'
+- '8'
```

Comment:
For some reason this is on all our travis-ci files 😄 

You only need one of those values so you can do it either like this:
```
node_js:
- '8'
```

or you could also make it reference a .nvmrc file – we use this in management interface as our team uses nvm anyway.
`echo '8.9.0'>.nvmrc` (commit this file)
`nvm install`
`nvm use`

and now in travis ci, remove the `node_js` block completely

Reply:
Ah yes, I just happen to want to test against all the different node versions out there, and not any specific one, so that also explains why I explicitly put the node version in the deploy down there. Still, Travis' node setups are kinda all acting differently???

Anyway, thanks for the tip! I'll probably address this in a separate PR (if I get around to it :P)